### PR TITLE
Add a systray menu to show all the times at a glance

### PR DESCRIPTION
**Pull request by andydotxyz**
Just as it sounds. Plus we can show the window which will now hide instead of exiting the app...

Screenshot 2022-08-24 at 22 38 06

### CODE REVIEW

**Comment by andydotxyz** [comment]
Alternative layout that could be used which feels neater: 
Screenshot 2022-08-24 at 22 47 49

**Comment by Bluebugs** [comment]
I have a preference for the second layout. Much more clean.

**Comment by andydotxyz** [comment]
Updated to other layout :)

File: main.go [hunk 1]
```diff
@@ -29,6 +32,44 @@ func main() {
...
+			setupSystrayMenu(a, w, store)
+		}
+	}()
+}
+
+func setupSystrayMenu(a desktop.App, w fyne.Window, store *cityStore) {
+	times := make([]*fyne.MenuItem, len(store.list)+2)
+	times[0] = fyne.NewMenuItem("Show Nomad", func() {
+		w.Show()
+	})
```
Comment (Jacalz) [inline main.go:61]: You don't need to wrap the function call. You can just do the following:
```suggestion
	times[0] = fyne.NewMenuItem("Show Nomad", w.Show)
```

File: main.go [hunk 2]
```diff
@@ -29,6 +32,44 @@ func main() {
 	w.Resize(fyne.NewSize(300, 500))
 	w.SetIcon(resourceIconPng)
 
+	if deskApp, ok := a.(desktop.App); ok {
+		w.SetCloseIntercept(func() {
+			w.Hide() // don't close the window if system tray used
+		})
```
Comment (Jacalz) [inline main.go:38]: You don't need to wrap the function call. You can just do the following:
```suggestion
		w.SetCloseIntercept(w.Hide) // don't close the window if system tray used
```

**Review by Jacalz — CHANGES_REQUESTED** [review]
I accidentally stumbled onto this PR. Added a small review :)

File: main.go (hunk 1 above)
Comment (andydotxyz) [reply main.go]: Oh yes oops, sorry!

**Review by Jacalz — APPROVED** [review]
Looks good to me :)

**Comment by andydotxyz** [comment]
Maybe @Bluebugs or @itsjustdel could pop a :+1: as well?

**Review by itsjustdel — APPROVED** [review]

**Comment by itsjustdel** [comment]
This looks great! Very nice

**Pull request merged**